application/config/locales/shared/en.yml

```diff
@@ -22,6 +22,11 @@ en:
     success: "success"
     error: "error"
     cancelled: "cancelled"
+  events:
```

Comment:
I would put this into its own file under the folder events.
I think Rails will load all subfolders automatically